tools/ci/illegal_dme_files.py

```diff
@@ -0,0 +1,43 @@
+# Naive search for illegal files. Has no semantic knowledge, just a lexical
+# search for specific endings in a .dme
+#
+# This is basically a slightly edited verison of unticked_files.py.
+# Look there for precise documentation on the methods used here.
+from pathlib import Path, PureWindowsPath
+import argparse
+import sys
+
+INCLUDED_FILES = [
+    'paradise.dme'
+]
+
+ILLEGAL_FILES = (
```

Comment:
this is a string, not a list because parens need at least one comma to be considered a tuple i.e. `("foo") == "foo"` but `("foo",) != "foo"` (because the Python grammar is a trashfire). 

that's why you were able to get away with `....endswith(ILLEGAL_FILES)`; if you turn this into a proper list you'll need to iterate over each item in the list too.

Reply:
.endswith accepts tuples of strings, .startswith accepts them also